doc/source/contributing/development_plan.rst

```diff
@@ -1,45 +1,51 @@
+by providing easy ways to manipulate ICESat-2 data in the appropriate input type required by each library.
+Workflows showcasing complex analyses to answer pressing science questions provide an opportunity for new reseachers to build on existing work.
 
 Validation and Integration with Other Products
-----------------------------------------------
-The complexity of multiple data access systems, many with different metadata formats and API access types, presents a challenge for finding and integrating diverse datasets to construct long time series. Many open-source resources provide tools for manipulating certain types of datasets, but few collate these resources into one computing environment (see `Improving Accessibility to Advanced Computing`_) and provide wrappers and examples to easily conduct frequently performed analysis tasks. This portion of the development plan, driven by researcher use cases, will combine existing resources with new ones to improve researcher ability to easily compare diverse datasets across varying sensor types and spatial and temporal scales.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The complexity of multiple data access systems, many with different metadata formats and API access types, 
+presents a challenge for finding and integrating diverse datasets. 
+Driven by researcher use cases, icepyx contains a framework for easily adding a new product/sensor to an existing data analysis pipeline,
+improving researcher ability to easily compare diverse datasets across varying sensor types and spatial and temporal scales.
 
 Modifying the Development Plan
 ------------------------------
-Everyone is invited to review and propose new items for the Development Plan. icepyx is continually evolving and its direction is driven by your feedback and contributions.
-
-Items listed in the Development Plan should be brief summaries of more detailed proposals. Each item listed should include:
-
-1. Summary of proposed changes/additions
-2. Motivation for the changes
-3. Statement describing how the changes fit within the icepyx scope
-4. More detailed plan for changes (e.g. implementation plan, examples (even if not implemented), potential issues)
+Everyone is invited to review and propose new themes for the Development Plan.
+icepyx is continually evolving and its direction is driven by your feedback and contributions.
 
-Please submit your proposal as a GitHub issue, which will provide the developers and community members an opportunity to provide feedback on your suggestion. Once there is agreement on the proposal, submit a pull request to update the Development Plan, including a link to the discussion issue.
\ No newline at end of file
+If you'd like to add a theme to this development plan, please submit your idea as a GitHub issue to solicit community feedback.
+Once there is agreement on your idea, submit a pull request to update the Development Plan, including a link to the discussion issue.
```

Comment:
Wondering if there's value with enabling the 'Discussions' tab and migrating some of the https://github.com/icesat2py/icepyx/labels/Discussion%20-%20feedback%20solicited issues into there?

On one hand, it's yet another thing to manage, but on the other hand, we could separate long-winded discussion points from issues that are usually of a more short-term and technical nature (e.g. bug reports and feature requests).

Reply:
I've gone back and forth on this since we were using Pangeo's Discourse. I think if we enable discussions (which didn't exist then) we'd want to direct everything there, which could actually be good for maintenance because it's all in one place (and as you say, separates discussion from to-dos better).

Reply:
Hmm, or we could just provide a link to https://discourse.pangeo.io/c/science/icesat-2/16? To be honest, the discourse forum isn't very active, and I'm not sure if using GitHub discussions would magically create more discussions. Leave it up to you to make that call.

Reply:
Discussions activated and discussion issues moved. We'll have to add a note and link to the contact us page.